examples/ExactlyOnceBasic/Program.cs

```diff
@@ -0,0 +1,680 @@
+                cancelled.Cancel();
+            };
+            AppDomain.CurrentDomain.ProcessExit += (_, e) =>
+            {
+                cancelled.Cancel();
+                completed.Token.WaitHandle.WaitOne();
+            };
+
+            if (args.Length != 2)
+            {
+                Console.WriteLine("Usage:");
+                Console.WriteLine("create_words <transactional-id>");
+                Console.WriteLine("reverse_words <transactional-id>");
+                completed.Cancel();
+                return;
+            }
+
+            var command = args[0];
+            var transactionalId = args[1];
+
+            IConfiguration configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("./appsettings.json")
+                .Build();
+
+            var producerConfig = configuration.GetSection("Producer").Get<ProducerConfig>().ThrowIfContainsNonUserConfigurable();
+            var consumerConfig = configuration.GetSection("Consumer").Get<ConsumerConfig>().ThrowIfContainsNonUserConfigurable();
+            var generalConfig = configuration.GetSection("General");
+            var wordsConfig = generalConfig.GetSection("Words");
+            var reversedWordsConfig = generalConfig.GetSection("ReversedWords");
+
+            var wordsTopicName = wordsConfig.GetValue<string>("TopicName");
+            var wordsTopicPartitions = wordsConfig.GetValue<int>("TopicPartitions");
+
+            var reversedWordsTopicName = reversedWordsConfig.GetValue<string>("TopicName");
+            var reversedWordsTopicPartitions = reversedWordsConfig.GetValue<int>("TopicPartitions");
+
+            var adminClient = new AdminClientBuilder(producerConfig).Build();
+            await CreateTopic(adminClient, wordsTopicName, wordsTopicPartitions);
+            await CreateTopic(adminClient, reversedWordsTopicName, reversedWordsTopicPartitions);
+
+            producerConfig.TransactionalId = transactionalId;
+            var consumerBuilder = new ConsumerBuilder<Null, string>(consumerConfig);
+            var producerBuilder = new ProducerBuilder<Null, string>(producerConfig);
+            var retryInterval = generalConfig.GetValue<TimeSpan>("RetryInterval");
+            var localTransactionOperationTimeout = generalConfig.GetValue<TimeSpan>("LocalTransactionOperationTimeout");
+
+            switch (command)
+            {
+                case "create_words":
+                    new WordsCreator
+                    {
+                        OutputTopic = wordsTopicName,
+                        ProduceRate = wordsConfig.GetValue<TimeSpan>("ProduceRate"),
+                        RetryInterval = retryInterval,
+                        LocalTransactionOperationTimeout = localTransactionOperationTimeout,
+                        ProducerBuilder = producerBuilder,
+                        Log = Program.Log,
```

Comment:
I think `Logger` is a better name for this property.

Reply:
I've called it Log because this is not a complete logger, it's an action that corresponds to the Program method.
It seems better to call `Log("message")`  than `Logger("message")`.